### DISCUSSION TITLE
how to fix this error

### DISCUSSION THREAD

**Discussion by DeedDaring** [termux/termux-packages#15575] [Q&A] [answered]
![Screenshot_20230310-135150](https://user-images.githubusercontent.com/126764204/224262290-e2b6eaff-c4da-43cb-9a8e-111977f14b1e.jpg)

**Comment by DeedDaring** [comment]
Please response me

**Comment by sylirre** [comment, marked as answer]
Just don't run `pkg` as root.

The package manager has hardcoded check for root user id and it can't be disabled. We added it intentionally as package manager in Termux **is not** supposed to be used under root due to unwanted side effects which will happen otherwise.
